spec/schemas/_common.yaml

```diff
@@ -1152,7 +1152,7 @@ components:
         refresh_size_lag:
           $ref: '#/components/schemas/RemoteStoreUploadRefreshSizeLagStats'
         total_time_spent:
-          description: The total amount of time, spent on uploads to the remote segment store.
+          description: The total amount of time spent on uploads to the remote segment store.
```

Comment:
"The total amount of time spent uploading [noun] to the remote segment store"?

Reply:
Upload actions?